gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -59,6 +59,9 @@
   public static final long DEFAULT_JOB_CONFIG_FILE_MONITOR_POLLING_INTERVAL = 300000;
   // Directory where all job configuration files are stored
```

Comment:
Please modify this comment to specify this is assumed to be in local file system.